### PR TITLE
A few points

### PR DESCRIPTION
**Pull request by vrashabhirde**
1. Made the titles first person so it can be better internalized (and for consistency)
2. Added a point and example around Disagree and Commit
3. Some punctuation OCD

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -24,38 +24,39 @@ This page is in constant WIP, please refer to it regularly.
...
   
   * We value simple and clear solutions that solve problems without unnecessary complexity. At the same time we keep the long term vision in mind and don’t just build for today. 
     
 
-__Outside in ownership__
+__We employ Outside-In ownership__
   
-  * We value shipping early and observing in production; we are fearless. We believe a developer is responsible for owning their work all the way through to delivering value, including monitoring and performance. 
+  * We value shipping early and observing in production.
+  * We are Fearless; We believe a developer is responsible for owning their work all the way through to delivering value, including monitoring and performance.
```
Comment (redroot) [inline readme.md]: Fearless is interesting, does that come from any specific blogpost/manifesto that's worth linking to? I agree with the idea, but I normally define this as `We value ownership`, maybe fearless ownership?

Comment (vrashabhirde) [reply readme.md]: It was in the original one, I thought it was interesting too and was too buried so just called it out seperately

**Comment by akash** [comment]
I kind of like the "we do xxx" style, sounds more affirming when reading it

**Comment by soulnafein** [comment]
I'm fine with either, but the original group decided against it. If we want to change it I'd like to see at least 10 engineers liking it.

File: readme.md (hunk 1 above)
Comment (CGA1123) [reply readme.md]: I like pointing out ownership/responsibility.
I’m not clear on how that amounts to fearlessness though - could you clarify @vrashabhirde?

File: readme.md [hunk 2]
```diff
@@ -65,33 +66,34 @@ __We strive to be informed by data as much as possible__
...
   
   * We practice shared code and project ownership. Any team member should be able to work on any project interchangeably and we aim to rotate teams to ensure people have exposure to all parts of the codebase.
 
 
-
-__Healthy conflict__
+__We encourage Healthy conflicts__
   
   * We believe in a blameless culture; which includes holding each other accountable through respectful feedback
+  * We strongly believe in the principle of "Disagree and Commit". We are tenacious and make strong but respectful arguments. However, once a decision is determined, we commit to it completely in belief and execution
```
Comment (CGA1123) [inline readme.md]: I disagree with committing completely in belief. I think it’s fine to continue to disagree with a decision as long as it isn’t significantly impacting performance - and if it is then maybe the reasoning behind a decision is not clear enough to the individual. Having an extra discussion is more worth it than ‘expecting’ commitment.

I want to avoid this clause sounding a bit like “every team decision is final and you’re not allowed to argue it further” - at the same time I’m not disagreeing with the fact that compromises have to made.

File: readme.md (hunk 1 above)
Comment (PChambino) [reply readme.md]: I think the fearless part was meant to the previous point:
`We value shipping early and observing in production. We are Fearless.`

Comment (nexusventuri) [reply readme.md]: I think having it in the previous section is better, now the first point is less memorable?

File: readme.md (hunk 2 above)
Comment (Arsen7) [reply readme.md]: The movie "12 Angry Men" comes to my mind. In our case it's much easier to make a verdict, but there is still a possibility that the majority was wrong.

File: readme.md (hunk 1 above)
Comment (tom-lord) [reply readme.md]: I like the word "Fearless" here. Although I think there's an overlap with `Confident` - which is one of the carwow core values.

Comment (plcstevens) [reply readme.md]: I would like to suggest a move away from fearless as it can be seen as uninclusive. Could we reword this towards how we tackle fear?

```
We fight fear; we believe in developer ownership and responsibility within a supportive, inclusive support network that allows them to deliver amazing value and grow themselves.
``` 

Or written better by someone else. I think this way we show ourselves as friendly to junior's and senior's and people who have struggled from being dropped in the deep end. Because Carwow doesn't do that, it fosters and supports its employees and we should show that in this document.

File: readme.md (hunk 2 above)
Comment (plcstevens) [reply readme.md]: I do think we have to make commitments that aren't always accepted by everyone, but perhaps we should alter this to handle changes in the future.

All disagreements should be noted for future reference and we will consider changing attitudes.

**Comment by Zeneixe** [comment]
Agree with @soulnafein , I think the added "We"s are redundant

File: readme.md (hunk 2 above)
Comment (tom-lord) [reply readme.md]: Further to Christian's point, I think this principle should be phrased as something like:

> We strongly believe in the principle of "Productive Disagreement". We are tenacious and make strong but respectful arguments. However, debates should not drag on and get in the way of productivity. Firm decisions should be made as promptly as possible, and - while not everyone may agree with the decision - fully committed to by all until an appropriate occasion to reopen the debate arises.

Comment (tom-lord) [reply readme.md]: > For example: You argued hard to change a KPI against a goal for your team. But a collective decision was taken against your argument. You now own the decision wholly and commit to it with your team. However, ongoing concerns can still be raised in an appropriate, time-boxed manner such as via OfficeVibe or team retrospectives.

Comment (plcstevens) [reply readme.md]: Yeah I can support that viewpoint.

Comment (JadeDickinson) [reply readme.md]: I'd support Tom's phrasing on this. In its current form, this principle is the same as one of Amazon's principles (https://www.amazon.jobs/en-gb/principles). We should be able to define our own culture.

File: readme.md (hunk 1 above)
Comment (akash) [reply readme.md]: I prefer that suggestion, it has the same sentiment while being more inclusive. Win-Win

Comment (CGA1123) [reply readme.md]: I relate the fearlessness more with shipping early and often, rather than with ownership and responsibility. Like what @PChambino said

i.e. 
- we are not afraid to try and test things.
- we believe in ownership and responsibility within a supportive framework
(in more eloquent words maybe)

**Comment by vrashabhirde** [comment]
All very good points made, thanks. I will summarize and make a new PR at some point. Excited to see passionate conversations around the principles.

File: readme.md (hunk 2 above)
Comment (vrashabhirde) [reply readme.md]: @JadeDickinson  👍 You got me there :)  
I was trying to get the sentiment across almost from a product building and product decisions perspective. I really like the language around product disagreements
